.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+| JIRA link | Slack thread (if any) | Product spec | Tech spec/One-pager | Bundle Size Difference |
+| -------- | --------------------- | ------------ | ------------------- | ---------------------- |
+| NA | NA | NA | NA | NA |
+
+Fixes # (issue)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
```

Comment:
This checklist can be more specific like 
Is your change backward compatible ?
Tested on major browsers (Chrome, Firefox, Safari, IE) ?
Tested in a consumer application ?